### DISCUSSION TITLE
How to send EJBCA container logs to Kibana

### DISCUSSION THREAD

**Discussion by aloeenmae** [Keyfactor/ejbca-ce#413] [Q&A] [answered]
Hi!

How can I configure EJBCA-CE container so, that it sends all its logs to Kibana server?

Thanks
Alo

**Comment by primetomas** [comment, marked as answer]
That would be a configuration in your container runtime, kubernetes or what you may be using. EJBCA sends to stdout, as best practice, and your runtime will be configured to consume the container log/output and send it wherever you want.

See for example https://sematext.com/guides/docker-logs/
